assets/javascripts/initializers/ai-helper.js

```diff
@@ -1,41 +1,85 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
 import i18n from "discourse-common/helpers/i18n";
+import AiComposerHelperMenu from "../discourse/components/ai-composer-helper-menu";
 import ModalDiffModal from "../discourse/components/modal/diff-modal";
+import { showComposerAiHelper } from "../discourse/lib/show-ai-helper";
 
-function initializeProofread(api) {
-  api.addComposerToolbarPopupMenuOption({
-    action: (toolbarEvent) => {
-      const modal = api.container.lookup("service:modal");
-      const composer = api.container.lookup("service:composer");
-      const toasts = api.container.lookup("service:toasts");
+function showErrorToast(api) {
```

Comment:
Super minor, can the `showErrorToast` helper live inside the initializer, that way we won't need to pass the whole `api` object to it every time it's invoked?